ocaml/xapi/xapi_globs.ml

```diff
@@ -919,8 +919,8 @@ let xenopsd_queues = ref ([
 
 let default_xenopsd = ref "org.xen.xapi.xenops.xenlight"
 
-let ciphersuites_good_outbound = ref None
-let ciphersuites_legacy_outbound = ref ""
+let ciphersuites_good_outbound = ref "!EXPORT:RSA+AES128-SHA256"
```

Comment:
Hi @lindig 
Thanks for your comments.
I just do as @thomassa  writes here: https://issues.citrite.net/browse/CA-222760?focusedCommentId=1400413&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1400413